api/src/main/java/org/ccci/idm/user/UserManager.java

```diff
@@ -277,7 +277,9 @@
      * @param queue              The {@link BlockingQueue} to add all users to.
      * @param includeDeactivated If <tt>true</tt> then deactivated accounts are included.
      * @return number of users enqueued
+     * @deprecated Since 1.0.0, use {@link UserManager#streamUsers(boolean)} instead.
```

Comment:
I want to hang on to the method for the time being. But this reminds me that I should remove some other deprecated code in a subsequent PR.

Reply:
ok. What is using it?

Reply:
I believe just The Key, which we are refactoring. but it will provide a quick path back if we find that it still needs to use a `BlockingQueue` for processing.

This library is used by several other identity related services, but I don't believe any of those use the enqueue method.